.circleci/config.yml

```diff
@@ -19,11 +19,6 @@ jobs:
         root: .
         paths:
         - ./cluster-operator
-    - deploy:
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
```

Comment:
This won't do anything because this should be never merged to master but just for sanity.